metadata.rb

```diff
@@ -9,5 +9,6 @@
 recipe 'master', 'Installs a Jenkins master'
 
 depends 'apt',   '~> 2.0'
+depends 'logrotate'
```

Comment:
You probably want to put at least some version constraint on the logrotate cookbook to keep it from changing radically under you.